packages/dev/docs/pages/blog/building-a-combobox.mdx

```diff
@@ -0,0 +1,94 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {BlogPostLayout} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+keywords: [combobox, accessibility, mobile, react spectrum, react, spectrum, interactions, touch]
+description: In this post we take a deeper look into some of the challenges faced when building an accessible and mobile friendly ComboBox.
+date: 2020-08-25
+author: '[Daniel Lu](https://github.com/LFDanLu)'
+---
+
+# Building a ComboBox
+
+## Intro
+
+As of INSERT DATE HERE, React Spectrum ComboBox has been released as an alpha. For those who may be unfamiliar with a combobox, it is an input field that has a popover menu associated with it. The popover menu contains
```

Comment:
I figured this would go up after combobox releases as an alpha, hence the placeholder.